firmware/examples/Dac/project_config.hpp

```diff
@@ -0,0 +1,10 @@
+// This file overrides the default configuration options in the
```

Comment:
I'm going to go through the examples and remove this project_config.hpp file. All of them do not depend on custom modification in this file, therefor the are redundant.

Reply:
Please remove this from your example Dac.